sunpy/image/tests/test_transform.py

```diff
@@ -318,7 +318,7 @@ def test_nans(rot30):
         axs[i, 0].imshow(image_with_nans, vmin=-1.1, vmax=1.1)
         for j in range(6):
             if j not in _rotation_registry[method].allowed_orders:
-                with pytest.raises(ValueError):
+                with pytest.raises(ValueError, match="!!"):
```

Comment:
This one has not been updated, you can get this message from the figure tests that are run on circleCI and is at the bottom of the pull request.

Reply:
When I ran the tests mentioned by the issue's author, I didn't get any flags, neither in vscode nor while running the test through the terminal. I think I might have missed this because even without fixing this all tests under tox -e py312 are passing, or maybe this was skipped. I will fix this.

Reply:
That is because this is a figure test, its not run normally but only under specific conditions which the tox command does not use by default.

This is why you will need to look at the circleCI test.

Reply:
I understand. Thank You.

Reply:
Made the changes. Thank You.